qiskit/execute_function.py

```diff
@@ -69,6 +69,7 @@ def execute(
     meas_map=None,
     scheduling_method=None,
     init_qubits=None,
+    use_measure_esp=None,
```

Comment:
Is it possible to set this as a run_config argument, or must this be passed explicitly to `assemble`? Perhaps we could ask for this to be set in `run_config` and pulse this out to place it in `assemble`? This way we keep the `execute` signature generic to any backend. In the long run `assemble` will be moved into the compiler so changes there are Ok.

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6543/commits/9af1534eb4eb37589a5ecef10a0521ff0d3ecc86.